pkgs/top-level/all-packages.nix

```diff
@@ -22509,6 +22509,16 @@ in
   way-cooler = throw ("way-cooler is abandoned by its author: " +
     "https://way-cooler.org/blog/2020/01/09/way-cooler-post-mortem.html");
 
+  wf-config = callPackage ../applications/window-managers/wayfire/wf-config.nix { };
+
+  wayfireApplications-unwrapped = callPackage ../applications/window-managers/wayfire/applications.nix { };
+  wayfirePlugins = callPackage ../applications/window-managers/wayfire/plugins.nix {
+    inherit (wayfireApplications-unwrapped) wayfire;
+  };
+
+  wayfireApplications = wayfireApplications-unwrapped.withPlugins (plugins: [ plugins.wf-shell ]);
+  inherit (wayfireApplications) wayfire wcm;
+
```

Comment:
I would prefer these to be sorted alphanumerical. Maybe like this:

```
wayfireApplications = wayfireApplications-unwrapped.withPlugins (plugins: [ plugins.wf-shell ]);
inherit (wayfireApplications) wayfire wcm;
wayfireApplications-unwrapped = callPackage ../applications/window-managers/wayfire/applications.nix { };
wayfirePlugins = callPackage ../applications/window-managers/wayfire/plugins.nix {
  inherit (wayfireApplications-unwrapped) wayfire;
};
wf-config = callPackage ../applications/window-managers/wayfire/wf-config.nix { };
```
Also: How does this work without defining `wcm` as a package?

Reply:
> Also: How does this work without defining `wcm` as a package?

```nix
{ inherit (wayfireApplications) wayfire wcm; }
```

is equivalent to

```nix
{ wayfire = wayfireApplications.wayfire;
  wcm = wayfireApplications.wcm; }
```

Reply:
I’ll resort alphabetically :)